### PR TITLE
[Module 2] Basics. Node.js fundamental theory.

### PR DESCRIPTION
**Pull request by ReshetovItsMe**

### CODE REVIEW

File: Module 2. Basics. Node.js fundamental theory./task2/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+const csv = require('csvtojson');
+const fs = require('fs');
+const { pipeline } = require('stream');
+
+const inputFilePath = process.argv[2]; // input file from npm task argument
+const outputFilePath = process.argv[3]; // input file from npm task argument
+
+pipeline(fs.createReadStream(inputFilePath).pipe(csv()), fs.createWriteStream(outputFilePath),
```
Comment (unfalse) [inline Module 2. Basics. Node.js fundamental theory./task2/index.js]: Are you sure this code will load the csv file line by line as suggested in evaluation criteria? (the file is not loaded fully in the RAM)

Comment (ReshetovItsMe) [reply Module 2. Basics. Node.js fundamental theory./task2/index.js]: yes, I think it's hint in our evaluation criteria, they recommend to use pipeline for this criteria.

Comment (unfalse) [reply Module 2. Basics. Node.js fundamental theory./task2/index.js]: how are you going to prove that file isn't loading fully into RAM?

Comment (ReshetovItsMe) [reply Module 2. Basics. Node.js fundamental theory./task2/index.js]: changed to byline stream

File: Module 2. Basics. Node.js fundamental theory./package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "name": "learn-nodejs-hw",
+  "version": "1.0.0",
+  "description": "This is project for home tasks which were made in EPAM NodeJS course",
+  "main": "index.js",
+  "scripts": {
+    "task1": "nodemon ./task1/index.js",
+    "task1rl": "nodemon ./task1/index-rl.js",
+    "task2": "nodemon ./task2/index.js ./task2/nodejs-hw1-ex1.csv ./task2/books.txt",
+    "task2pipe": "nodemon ./task2/index-pipe.js ./task2/nodejs-hw1-ex1.csv ./task2/books.txt",
+    "task3": "echo \"Error: no test specified\" && exit 1"
```
Comment (unfalse) [inline Module 2. Basics. Node.js fundamental theory./package.json]: remove task3 please

File: Module 2. Basics. Node.js fundamental theory./task2/books.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{"Book":"The Compound Effect","Author":"Darren Hardy","Amount":"5","Price":"9.48"}
+{"Book":"The 7 Habits of Highly Effective People","Author":"Stephen R. Covey","Amount":"4","Price":"23.48"}
+{"Book":"The Miracle Morning","Author":"Hal Elrod","Amount":"10","Price":"21.34"}
+{"Book":"Influence: The Psychology of Persuasion","Author":"Robert B. Cialdini","Amount":"4","Price":"12.99"}
+{"Book":"The ONE Thing","Author":"Gary Keller","Amount":"1","Price":"11.18"}
```
Comment (unfalse) [inline Module 2. Basics. Node.js fundamental theory./task2/books.txt]: I don't think we need the output file in repo

**Comment by unfalse** [comment]
overall is ok

File: Module 2. Basics. Node.js fundamental theory./task2/books.txt (hunk 3 above)
Comment (ReshetovItsMe) [reply Module 2. Basics. Node.js fundamental theory./task2/books.txt]: I think it's good for result example. But ok I'll delete it if you want

**Comment by ReshetovItsMe** [comment]
@unfalse everything is fixed

**Pull request merged**